content/200-concepts/100-components/01-prisma-schema/07-names-in-underlying-database.mdx

```diff
@@ -101,9 +85,12 @@ Custom constraint and index names are currently not supported by MongoDB.
 
 </Admonition>
 
+
+When introspecting a database, the `map` argument will _only_ be rendered in the schema the name differs from Prisma's [default constraint naming convention for indexes and constraints](#prismas-default-naming-conventions-for-indexes-and-constraints).
+
 <Admonition type="alert">
 
-If you use Prisma Migrate and want to maintain your existing constraint and index names, **do not** immediately run `prisma migrate` or `prisma db push`. This will **change any underlying constraint name that does not follow Prisma's convention**. Follow the [upgrade path for the Prisma Migrate workflow](/guides/upgrade-guides/upgrading-versions/upgrading-to-prisma-3/named-constraints#upgrade-paths-for-prisma-migrate-users).
+If you use Prisma Migrate and want to maintain your existing constraint and index names, **do not** immediately run `prisma migrate` or `prisma db push`. This will **change any underlying constraint name that does not follow Prisma's convention**. Follow the [upgrade path for the Prisma Migrate workflow](../../../guides/upgrade-guides/upgrading-versions/upgrading-to-prisma-3/named-constraints#upgrade-paths-for-prisma-migrate-users).
```

Comment:
```suggestion
If you use Prisma Migrate and want to maintain your existing constraint and index names, **do not** immediately run `prisma migrate` or `prisma db push`. This will **change any underlying constraint name that does not follow Prisma's convention**. Follow the [upgrade path for the Prisma Migrate workflow](/guides/upgrade-guides/upgrading-versions/upgrading-to-prisma-3/named-constraints#upgrade-paths-for-prisma-migrate-users).
```